crates/bevy_pbr/src/render/pbr_functions.wgsl

```diff
@@ -33,9 +33,8 @@
 #endif
 
 
-// Biasing info needed to sample from a texture when calling `sample_texture`.
-// How this is done depends on whether we're rendering meshlets or regular
-// meshes.
+// Biasing info needed to sample from a texture. How this is done depends on
+// whether we're rendering meshlets or regular meshes.
 struct SampleBias {
```

Comment:
Not for this PR, but as a side note the derivative stuff for meshlet meshes are not biases. We should probably rename this type to TextureSampleInfo in the future or something.